### PR TITLE
Docs: Setup directories with correct permissions

### PR DESCRIPTION
**Pull request by vagishagupta23**
Documentation updated with the steps to setup the above directories
with the correct permissions.

Make sure these boxes are signed before submitting your Pull Request
-- thank you.

- [x] I have read the contributing guide lines at
  https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation
  contribution agreement at
  https://suricata-ids.org/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the
  changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/3038

### CODE REVIEW

File: doc/quickstart.rst [hunk 1]
```diff
@@ -52,8 +52,37 @@ One option is to simply run ``suricata-update`` as root or with
...
+
+    sudo usermod -a -G suricata username
+
+Verify whether group has been changed::
+
+    ls -al /etc/suricata
+    ls -al /var/lib/suricata/rules
+    ls -al /var/lib/suricata/update
+
+Reboot your system. Run ``suricata-update`` without a sudo to check
```
Comment (shivan1b) [inline doc/quickstart.rst]: Do we need a reboot?

File: doc/quickstart.rst [hunk 2]
```diff
@@ -52,8 +52,37 @@ One option is to simply run ``suricata-update`` as root or with
...
+Next, change the group of the directories and its files recursively::
+
+    sudo chgrp -R suricata /etc/suricata
+    sudo chgrp -R suricata /var/lib/suricata/rules
+    sudo chgrp -R suricata /var/lib/suricata/update
+
+Setup the directories with the correct permissions for the ``suricata``
+group::
+
+    sudo chmod -R g+r /etc/suricata/
```
Comment (shivan1b) [inline doc/quickstart.rst]: Hmm most of the configuration like update.yaml, modify.yaml goes here so it should not just have read access.

File: doc/quickstart.rst (hunk 1 above)
Comment (vagishagupta23) [reply doc/quickstart.rst]: @shivan1b It doesn't run on my system without a reboot.

File: doc/quickstart.rst (hunk 2 above)
Comment (vagishagupta23) [reply doc/quickstart.rst]: So, it should have both read and write access? Then, there should also be a change in the docs where it is written `Directory /etc/suricata: read access`. Should I consider updating it?

**Comment by vagishagupta23** [comment]
> @vagishagupta23 Minor changes requested. Apart from this, please note that we should probably not use `/etc/suricata` and `/var/lib` for example, since it could be any other path that user defines during configuration and is by default `/usr/local`. Maybe, add a note that people should check their configuration directory and use that accordingly?

Yes, I was also confused about it. Thanks for confirming. Will address these changes.

> This probably won't be a problem with experienced Suricata users but we should write the docs such that beginners do not end up making a lot of mistakes.
> 
> Looks awesome other than this. Thanks! :)

**Comment by vagishagupta23** [comment]
A new PR is here #168.

**Pull request closed without merging**